examples/external-data/jest.config.cjs

```diff
@@ -3,35 +3,11 @@
  * Licensed under the MIT License.
  */
 
-// Get the test port from the global map and set it in env for this test
-const testTools = require("@fluidframework/test-tools");
+const getBaseJestConfig = require("../../getBaseJestConfig.cjs");
 const { name } = require("./package.json");
-
-const mappedPort = testTools.getTestPort(name);
-process.env["PORT"] = mappedPort;
-
-module.exports = {
-	preset: "jest-puppeteer",
-	globals: {
-		PATH: `http://localhost:${mappedPort}`,
-	},
-	testMatch: ["**/?(*.)+(spec|test).[t]s"],
-	testPathIgnorePatterns: ["/node_modules/", "dist"],
-	transform: {
-		"^.+\\.ts?$": "ts-jest",
-	},
-	moduleNameMapper: {
-		// Remove explicit .js from local paths to allow jest to find the .ts* files
-		"^(\\.{1,2}/.*)\\.js$": "$1",
-	},
-	reporters: [
-		"default",
-		[
-			"jest-junit",
-			{
-				outputDirectory: "nyc",
-				outputName: "jest-junit-report.xml",
-			},
-		],
-	],
+const config = getBaseJestConfig(name);
+config.moduleNameMapper = {
+	// Remove explicit .js from local paths to allow jest to find the .ts* files
+	"^(\\.{1,2}/.*)\\.js$": "$1",
```

Comment:
This one and the one under `examples/service-clients/azure-client/external-controller/` had this non-standard `moduleNameMapper`... not sure at all if it's really required, but leaving them be for now.

Reply:
Can you file a bug? I don't see why these packages would special. cc: @ChumpChief @jason-ha

Reply:
I can only assume it was carryover from some copy paste and/or an artifact of external-controller living apart from most of our examples for a while.  I also don't think they should be special.

Reply:
I was able to remove it from the `azure-client/external-controller` jest config but found out why this one needs it. Its test files import from modules in `src/` and have to specify .js extensions in those imports (almost none of our other jest tests have imports like that; the ones that do have their own jest configs that don't leverage the common one) and since we're using `ts-jest` (because of the lack of full support for ESM in Jest), those imports fail unless we do this module name mapping.

I'm going to move it to the common config with a note about why it's there.

Reply:
I also created https://dev.azure.com/fluidframework/internal/_workitems/edit/26461 to look at those non-standard jest configs at some point.